js/google_ima.js

```diff
@@ -147,6 +147,7 @@ require("../html5-common/js/utils/utils.js");
         this.adPlaybackStarted = false;
         this.vcPlayRequested = false;
         this.savedVolume = -1;
+        this.useGoogleAdUI = false;
```

Comment:
Sorry quick q though - don't we want to use Google's UI elements by default and only use ours if if specified?  or does this mean to use Ooyala UI for google ads?  I think the name confused me